packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -31,7 +33,7 @@ abstract class RunCommandBase extends FlutterCommand {
           'Note: this flag will be removed in favor of the --debug/--profile/--release flags.');
 
     argParser.addFlag('trace-startup',
-        negatable: true,
+        negatable: false,
```

Comment:
We should leave it as negatable. I was about to make `flutter drive` to default to `true` with an option of turn it off. This is because when you run benchmarks you almost never want this flag to be off.